apps/web/src/components/organisms/EventCalendar/EventCalendarItem.tsx

```diff
@@ -175,26 +165,13 @@ interface EventTheme {
 }
 
 function getEventTheme(event: Event, isActive: boolean): EventTheme {
-  const config = EVENT_TYPE_CONFIG[event.type as EventType]
-
-  if (!config) {
-    return {
-      item: {
-        base: `${DEFAULT_STYLES.itemBg} ${DEFAULT_STYLES.itemBorder} ${DEFAULT_STYLES.itemText}`,
-        gradient: DEFAULT_STYLES.itemBgGradientFade,
-      },
-      card: {
-        bg: DEFAULT_STYLES.cardBg,
-        text: DEFAULT_STYLES.cardText,
-        borderHover: DEFAULT_STYLES.cardBorderHover,
-      },
-    }
-  }
+  // Use OTHER config as fallback if event type is not found
+  const config = EVENT_TYPE_CONFIG[event.type as EventType] || EVENT_TYPE_CONFIG.OTHER
 
   if (isActive) {
     return {
       item: {
-        base: `${config.colors.itemBg} ${config.colors.itemBgHover} ${config.colors.itemBorder} ${config.colors.itemText}`,
+        base: `${config.colors.itemBg} ${config.colors.itemBorder} ${config.colors.itemText}`,
```

Comment:
You should use `cn` here.

```suggestion
        base: cn(config.colors.itemBg, config.colors.itemBorder, config.colors.itemText),
```